test/plugin/in_tail/test_position_file.rb

```diff
@@ -196,7 +196,11 @@ def build_files(file)
       @file.seek(0)
       lines = @file.readlines
       assert_equal 2, lines.size
-      assert_equal "valid_path\t000000000000000a\t000000000000000b\n", lines[0]
+      if Fluent.windows?
+        assert_equal "valid_path\t0000000000000000a\t000000000000000b\n", lines[0]
```

Comment:
[question] Why it happens?

Reply:
I'll dig it.

Reply:
Aha! I got it.
In Windows, it is also needed to use binmode for Tempfile! :punch: